### DISCUSSION TITLE
Creating a new project is running endless

### DISCUSSION THREAD

**Discussion by kausikMR** [appwrite/appwrite#3262] [Q&A]
From Appwrite v0.14 I couldn't create a new project

<img width="934" alt="Screenshot 2022-05-21 at 9 50 18 AM" src="https://user-images.githubusercontent.com/74778433/169635130-8cc3a720-fe90-4d65-bfbd-725f155fc609.png">

**Comment by Meldiron** [comment]
Hey there 👋 Project creation usually takes only a few seconds. Could you please try it again and share logs with me? To get logs, you can use the command `docker compose logs appwrite`.

**Reply by kausikMR** [reply]
I just cleared all the containers and images in my Docker and hosted the latest version 0.14.2 again, now the project creation works fine.
